astropy/convolution/tests/test_discretize.py

```diff
@@ -74,21 +74,43 @@ def test_pixel_sum_2D(model_class, mode):
 def test_gaussian_eval_2D(mode):
     """
     Discretize Gaussian with different modes and check
-    if result is at least similar to Gaussian1D.eval()
+    if result is at least similar to Gaussian2D.eval()
     """
     model = Gaussian2D(1, 0, 0, 20, 20)
+
     x = np.arange(-100, 101)
     y = np.arange(-100, 101)
+
     x, y = np.meshgrid(x, y)
+
     values = model(x, y)
     disc_values = discretize_model(model, (-100, 101), (-100, 101), mode=mode)
-    assert_allclose(values, disc_values, atol=0.001)
+    assert_allclose(values, disc_values, atol=1e-3)
+
+@pytest.mark.skipif('not HAS_SCIPY')
+def test_gaussian_eval_2D_integrate_mode():
+    """
+    Discretize Gaussian with integrate mode
+    """
+    model_list = [Gaussian2D(1, 0, 0, 20, 20),
+                  Gaussian2D(1, 0, 0, 30, 20),
+                  Gaussian2D(1, 0, 0, 20, 30)]
+
+    x = np.arange(-100, 101)
```

Comment:
The array size is very large, which makes the test run very slow. It's probably ~few minutes on a standard desktop machine (I think the Travis-CI fail is related to that...).  Can reduce the array size here to 5x5 or even 3x3 and test against the `oversample` mode?